sys/events.js

```diff
@@ -27,13 +27,17 @@ EventService.prototype.emitEvent = function(hyper, req) {
             }
 
             return self.purger.purge(req.body.map(function(event) {
-                if (!event.meta || !/^\/\/[^\/]+/.test(event.meta.uri)) {
+                if (!event.meta || !event.meta.uri) {
                     hyper.log('error/events/purge', {
                         message: 'Invalid event URI',
                         event: event
                     });
                 } else {
-                    return event.meta.uri.toString();
+                    var uri = event.meta.uri.toString();
+                    if (!/^https?:/.test(uri)) {
```

Comment:
Nit-pick: while we have agreed to send proto-relative URIs to `/sys/event`, maybe it would be prudent to check for `//` as well. As it currently stands, if `en.wikipedia.org` is passed in, we'd end up with `http:en.wikipedia.org` which isn't what we expect to happen.